src/main/kotlin/frc/robot/subsystems/swerve/SwerveDrive.kt

```diff
@@ -178,6 +178,28 @@ class SwerveDrive private constructor
         estimator.resetPosition(pose.rotation, modulePositions, pose)
     }
 
+    private fun isSkidding(): Boolean {
+        var weightedAvg = 0.0
+        var denominator = 0.0
```

Comment:
Of what?
This code is not obvious at first glance. Please give better names and write some documentation.

Reply:
87aad34d6ab27dd65877815df2a6db47b6781c3f

Reply:
I still think it could be better, I at least would include the formula ChatGPT gave you (a link to the conversation with it is best).